RecoParticleFlow/PFProducer/python/particleFlowBlock_cff.py

```diff
@@ -9,17 +9,15 @@
 # include "RecoTracker/TrackProducer/data/CTFFinalFitWithMaterial.cff"
 from RecoParticleFlow.PFProducer.particleFlowBlock_cfi import *
 
-_phase2_hgcal_Importers = deepcopy(particleFlowBlock.elementImporters)
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 # kill tracks in the HGCal
-_phase2_hgcal_Importers[5].importerName = cms.string('GeneralTracksImporterWithVeto')
-_phase2_hgcal_Importers[5].veto = cms.InputTag('hgcalTrackCollection:TracksInHGCal')
-#_phase2_hgcal_Importers[2].source_ee = cms.InputTag('particleFlowSuperClusterHGCal')
-#_phase2_hgcal_Importers.append(
-#    cms.PSet( importerName = cms.string("HGCalClusterImporter"),
-#              source = cms.InputTag("particleFlowClusterHGCal"),
-#              BCtoPFCMap = cms.InputTag('particleFlowSuperClusterHGCal:PFClusterAssociationEBEE') ),
-#)
-_phase2_hgcal_Linkers = deepcopy(particleFlowBlock.linkDefinitions)
+phase2_hgcal.toModify(
+    particleFlowBlock,
+    elementImporters = { 5 : dict(
```

Comment:
the reason for not doing the clone and replace of the whole VPSet is that two different modifiers may need to both modify the same VPSet (as it happens in this case phase2_calo and phase2_timing).

It can be made to work also for other sequence types and not just VPSets, if there's interest in it (in fact it may be that it already works for them, I just didn't test it)